tests/assets.rs

```diff
@@ -31,8 +32,17 @@ fn all_themes_are_present() {
             "base16",
             "base16-256",
             "gruvbox",
+            "gruvbox-dark",
+            "gruvbox-dark-hard",
+            "gruvbox-dark-medium",
+            "gruvbox-dark-soft",
+            "gruvbox-hard",
             "gruvbox-light",
-            "gruvbox-white",
+            "gruvbox-light-hard",
+            "gruvbox-light-medium",
+            "gruvbox-light-soft",
+            "gruvbox-medium",
+            "gruvbox-soft",
```

Comment:
As there is no difference between `gruvbox` and `gruvbox-dark` and `gruvbox-dark-medium`, all three should not stay.
I vote to keep `gruvbox-dark-medium` , because it explains the difference between this and.. say, `gruvbox-dark-hard`. 
Whereas, users are going to have a hard time trying to differentiate between `gruvbox` and `gruvbox-dark-hard`
<br>

The same analogy applies to `gruvbox-light`, `gruvbox-hard`, `gruvbox-medium`, `gruvbox-soft`, etc..